dashboard/public/odometer/cdo-odometer.js

```diff
@@ -14,11 +14,16 @@ function Odometer(config) {
   // Create DOM elements.
   var odometer = $("<div class='odometer'>");
   for (var i = 0; i < config.digits; i++) {
+    // prettier-ignore
     var digit = $("<div class='digit'><div class='digit-current'>0</div><div class='digit-next'>1</div></div>");
     digits.push(digit);
     odometer.append(digit);
   }
-  $(config.parent).addClass('odometerParent').append(odometer).append("<div class='odometerGradient'>");
+  $(config.parent)
+    .addClass("odometerParent")
+    .append(odometer)
+    // prettier-ignore
+    .append("<div class='odometerGradient'>");
```

Comment:
out of curiosity, why do we need `prettier-ignore` here and above?

Reply:
To fix some of prettier errors that can't be fixed, here're some of them.


```
/Users/denyslevada/Projects/code-dot-org/dashboard/public/odometer/cdo-odometer.js
   1:10  error  'Odometer' is defined but never used                                                                                       no-unused-vars
   2:1   error  Delete `⏎`                                                                                                                 prettier/prettier
  22:19  error  Replace `.addClass('odometerParent').append(odometer)` with `⏎····.addClass("odometerParent")⏎····.append(odometer)⏎····`  prettier/prettier
  39:24  error  Replace `'.digit-current'` with `".digit-current"`                                                                         prettier/prettier
  40:24  error  Replace `'.digit-next'` with `".digit-next"`                                                                               prettier/prettier
  41:23  error  Replace `'top'` with `"top"`                                                                                               prettier/prettier
  57:37  error  Delete `·`                                                                                                                 prettier/prettier
  58:21  error  Replace `'top'` with `"top"`                                                                                               prettier/prettier
  64:29  error  Replace `'number'` with `"number"`                                                                                         prettier/prettier
✖ 9 problems (9 errors, 0 warnings)
```

p.s. talking about the one that you've left a comment on - prettier suggests to replace single quotes with double quotes, that'll break the js: .append("<div class='odometerGradient'>"); => .append("<div class="odometerGradient">");